readthedocs/oauth/services/github.py

```diff
@@ -450,11 +450,11 @@ def send_build_status(self, build, commit, state):
         owner, repo = build_utils.get_github_username_repo(url=project.repo)
 
         # select the correct state and description.
-        github_build_state = SELECT_BUILD_STATUS[state]['github']
-        description = SELECT_BUILD_STATUS[state]['description']
-        statuses_url = f'https://api.github.com/repos/{owner}/{repo}/statuses/{commit}'
+        github_build_state = SELECT_BUILD_STATUS[status]["github"]
+        description = SELECT_BUILD_STATUS[status]["description"]
+        statuses_url = f"https://api.github.com/repos/{owner}/{repo}/statuses/{commit}"
 
-        if build.state in BUILD_FINAL_STATES and build.success:
+        if status == BUILD_STATUS_SUCCESS:
```

Comment:
@humitos I think this is actually what we want? Basically just check if the status was success, then link to the built docs?

Reply:
Yeah, I think so 😅